movement/watch_faces/complication/discgolf_face.c

```diff
@@ -1,7 +1,7 @@
 #include <stdlib.h>
 #include <string.h>
-#include "discgolf_face.h"
-#include "watch.h"          // Remember to change number of courses in this file
+#include "discgolf_face.h"　　　　// Remember to change number of courses in this file
```

Comment:
```suggestion
#include "discgolf_face.h"    // Remember to change number of courses in this file
```

It seems there are non-ASCII characters. Is this intentional?